### PR TITLE
Giving RCEdit standard values but i used git wrong

### PR DESCRIPTION
**Pull request by vaporvee**
I changed things in the exporter so that the product name and description use the actual project name when there is no overwrite in the export settings

### CODE REVIEW

**Comment by vaporvee** [comment]
this issue is relevant for the pull request: #29569

**Review by Pukimaa — APPROVED** [review]
LGTM!

**Comment by vaporvee** [comment]
Its a really small change can it be put it into a possible 4.0.3? Or is the 4.1 milestone just standard procedure?

**Comment by aaronfranke** [comment]
@vaporvee Commits can be cherry-picked to stable branches only after they are first merged into master. It must go in the branch that will become 4.1 before it can go in 4.0.3.

**Comment by vaporvee** [comment]
Okay thanks thats my first commit here :)

File: platform/windows/export/export_plugin.cpp [hunk 1]
```diff
@@ -386,21 +386,26 @@ Error EditorExportPlatformWindows::_rcedit_add_data(const Ref<EditorExportPreset
 		args.push_back("--set-product-version");
 		args.push_back(product_version);
 	}
+	//Having a standard naming fallback preventing showing up as Godot Engine
+	args.push_back("--set-version-string");
+	args.push_back("ProductName");
+	if (product_name.is_empty())
+		args.push_back(GLOBAL_GET("application/config/name"));
+	else
+		args.push_back(product_name);
```
Comment (bruvzg) [inline platform/windows/export/export_plugin.cpp:395]: `GLOBAL_GET("application/config/name")` can be empty, so should be checked as well. Also, Godot code style require {} around all if blocks, use can use `clang-format` no automatically fix code style - see [Code style guidelines](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#c-and-objective-c).

```suggestion
	if (product_name.is_empty()) {
		String app_name = GLOBAL_GET("application/config/name");
		if (app_name.is_empty()) {
			args.push_back("Unnamed"); //Note: different platforms use diffrent names in this case: "Unnamed", "Godot Game".
		} else {
			args.push_back(app_name);
		}
	} else {
		args.push_back(product_name);
	}
```

Comment (vaporvee) [reply platform/windows/export/export_plugin.cpp:395]: there is stuff missing like making this for file description wich shows up in task manager so i'll implement it myself

Comment (vaporvee) [reply platform/windows/export/export_plugin.cpp]: does it look good to you now?

**Review by Pukimaa — APPROVED** [review]

**Comment by vaporvee** [comment]
I don't get what the error in static checks means

**Comment by akien-mga** [comment]
It means you have trailing whitespace on an empty line, which should be removed:

image

While fixing it, please also squash the commits. See [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase) for instructions.

**Comment by vaporvee** [comment]
okay should i just make a new pr at this point? 
grafik

**Comment by YuriSizov** [comment]
Superseded by https://github.com/godotengine/godot/pull/75705.

**Pull request closed without merging**